File: C++/1134.cpp
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{

    int gasolina, alcool, diesel, opcao;
    gasolina = 0;
    alcool = 0;
    diesel = 0;
    do
    {
        cin >> opcao;
        if (opcao == 2)
            gasolina = gasolina + 1;
        else if (opcao == 1)
            alcool = alcool + 1;
        else if (opcao == 3)
            diesel = diesel + 1;
    } while (opcao != 4);

    cout << "MUITO OBRIGADO\n";
    cout << "Alcool: " << alcool << "\n";
    cout << "Gasolina: " << gasolina << "\n";
    cout << "Diesel: " << diesel << "\n";
}

File: C#/1134.cs
using System;

class URI
{

    static void Main(string[] args)
    {

        int cliente = 5, alcool = 0, gasolina = 0, diesel = 0;


        while (cliente != 4)
        {
            cliente = Convert.ToInt32(Console.ReadLine());

            if (cliente == 1)
                alcool++;
            else if (cliente == 2)
                gasolina++;
            else if (cliente == 3)
                diesel++;
            else if (cliente == 4)
                ;
        }

        Console.WriteLine("MUITO OBRIGADO");
        Console.WriteLine("Alcool: {0}", alcool);
        Console.WriteLine("Gasolina: {0}", gasolina);
        Console.WriteLine("Diesel: {0}", diesel);



    }

}